yarn-project/circuit-types/src/logs/tx_l2_logs.ts

```diff
@@ -89,32 +89,6 @@ export abstract class TxL2Logs<TLog extends UnencryptedL2Log | EncryptedL2Log> {
   public equals(other: TxL2Logs<TLog>): boolean {
     return isEqual(this, other);
   }
-
-  /**
-   * Computes logs hash as is done in the kernel and app circuits.
-   * @param logs - Logs to be hashed.
-   * @returns The hash of the logs.
-   * Note: This is a TS implementation of `computeKernelLogsHash` function in Decoder.sol. See that function documentation
-   *       for more details.
-   */
-  public hash(logType: LogType = LogType.ENCRYPTED): Buffer {
```

Comment:
Removed this hash fn and added two separate hash fns for log type (encrypted or unencrypted) below.